src/components/GeneralInvoiceData/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "../../index.css";
+
```

Comment:
Again, in some places you're defining a function using `const` (which is fine), but you're mixing it with using `function` elsewhere. Just pick one and stick with it!